Remove inter-zone charge in firewall doc due to new announcement (#123186)
Azure/azure-content: articles/firewall/features.md

You can also associate Azure Firewall to a specific zone just for proximity reasons, using the service standard 99.95% SLA.

> There's no extra cost for a firewall deployed in more than one Availability Zone. Also, Microsoft has announced that Azure will not charge for the data transfer across availability zones regardless of using private or public IPs on your Azure resources [link](https://azure.microsoft.com/updates/update-on-interavailability-zone-data-transfer-pricing/) .

Reviewer: ```suggestion
There's no extra cost for a firewall deployed in more than one Availability Zone. Also, Microsoft has announced that Azure won't charge for the data transfer across availability zones, regardless of whether you use private or public IPs on your [Azure resources](https://azure.microsoft.com/updates/update-on-interavailability-zone-data-transfer-pricing/).
```
